Lombiq.HelpfulLibraries.Targets/Lombiq.HelpfulLibraries.Targets.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>

```

Comment:
Just a nitpick but could this detect the config from the csprojs or vice versa?

Reply:
All your projects currently have this.

If you added (as I did above for Latest) the values into a `Directory.Build.Props` that would resolve it.

I do this on my other projects for all the Orchard Versions as well, but didn't want to do that here, as it would confuse the pr (and possibly your parent solutions)

Reply:
Just in this project let's add this to a `Directory.Build.Props` then.

Reply:
Move to a Directory.Build.Props file in that folder